### DISCUSSION TITLE
Hello World

### DISCUSSION THREAD

**Discussion by POM-Developer** [iTwin/itwinjs-core#6226] [General]
Hello

New into this iTwin world.

Context to my question...

Bentley has a BDN network that gave us access to this iTwin,js.
https://bdn.bentley.com/

Also Bentley has a iTwin platform that we can subscribe
https://developer.bentley.com/

We are also Bentley customer, which give us access to manipulate iModel from the Connect or BIC portal, which from what I understand, is connect to Itwin in the backend.

If we use the SDK from the BDN to access the data of the itwin services ( example extract Forms/Issue/DM etc....). Do we need to be subscribe on the iTwin platform? 

Is my understanding is correct, we need to subscribe to the iTwin platform, only if we need to commercialize  a development, But if all we do is develop for internal usage (no commercial intent) the subscription to the itwin platform is not mandatory?

Just wanted to make sure we have the correct fondation to get start develop

Thank you

**Comment by tamanud** [comment]
Hi,

You have mentioned three different things here

1) iTwin.js - This is a open-source JavaScript library. No subscription is required to use this.

2) BDN (Bentley Developer Network) -  this gives access only to SDKs for different Bentley Desktop software products. 
    This does not include any cloud benefits (iTwin Services)

3) iTwin Platform Services - This is a paid subscription if you want to use Cloud services API. 
    We have a 3 month trial beyond which you need to buy a cloud credit based subscription.
